chainer/testing/parameterized.py

```diff
@@ -48,8 +48,11 @@ def _parameterize_test_case_generator(base, params):
         cls_name = _make_class_name(base.__name__, i, param)
 
         def __str__(self):
+            # Bind to a new variable.
+            param2 = param
```

Comment:
Do you have to nest this further e.g. in another function that constructs and return this `__str__` object?

Reply:
You are right.  I'll fix the code.